src/components/TopNav/ExportButton.tsx

```diff
@@ -59,7 +60,12 @@ export const ExportButton = () => {
     );
 
     zip.generateAsync({ type: 'blob' }).then(function (content) {
-      Mixpanel.track('export project downloaded', { projectId });
+      Mixpanel.track(DOWNLOAD_EVENT, { projectId });
+      if (window?.gtag) {
+        window?.gtag('event', DOWNLOAD_EVENT, {
+          proejctId: projectId,
```

Comment:
is it a typo, or it's intentional?
```suggestion
         projectId,
```